docs/sphinx/user_guide/feature/view.rst

```diff
@@ -82,6 +82,44 @@ accesses array entries with unit stride. The loop::
 
 access array entries with stride N :subscript:`n` * N :subscript:`(n-1)` * ... * N :subscript:`(j+1)`.
 
+MultiView
+^^^^^^^^^^^^^^^^
+
+A ``RAJA::MultiView`` object wraps an array-of-pointers,
+or a pointer-to-pointers, rather than a ``RAJA::View`` which wraps a single
+pointer or array. This allows a single ``RAJA::Layout`` to be applied to a
+large set of arrays, and encapsulates a set of arrays more easily.
+
+The instantiation of a MultiView works exactly like a standard View,
+except that it wraps an array-of-pointers::
+
+  int a1[4] = {5,6,7,8}; int a2[4] = {9,10,11,12}; // individual arrays of the same size
+  int * myarr[2]; // array of arrays which will be passed into MultiView
+  myarr[0] = a1; myarr[1] = a2;
+  RAJA::MultiView< int, RAJA::Layout<2> > MView(myarr, 2, 2);
+
+The MultiView can access individual arrays via the zero-th index::
+
+  MView( 0, 2, 2 ); // accesses 8, the (2,2)th element of the first array

```

Comment:
This comment would be clearer if they were changed to "access array entry at index (2,2), which has value 8. Same with the next one below.